### DISCUSSION TITLE
How to start for GSOC 2024

### DISCUSSION THREAD

**Discussion by Aman-Atar-002** [camicroscope/GSOC#59] [General]
Hi there I am Aman Atar currently planning to contribute to open source through GSOC and I found the camicroscope community quite intresting because of its cause of its features of assistance in the field of medicines through visualization. 
It would be really great if the community members would help me to know regarding how should I get started.

**Comment by birm** [comment]
We have a basic intro for new contributors here: https://camicroscope.org/contributing.html
Please let us know if you have any follow up questions!

**Comment by BasmaElhoseny01** [comment]
Hello I am Basma I am new to the community i was asking about how to begin to be able to join GSOC projects

**Comment by birm** [comment]
As a later update, we also have contributor tasks now! https://github.com/camicroscope/GSOC/blob/master/ContributorTasks.md

**Comment by malikrafsan** [comment]
Hi @birm and caMicroscope community👋👋

My name is Malik Akbar Hashemi Rafsanjani, a final-year computer science student from Indonesia (Bandung Institute of Technology)

I am very excited to contribute to this project for Google Summer of Code (GSoC) 2024. I am particularly interested in the caMicroscope as this organization helps healthcare through digitalization, and I am very interested in the digitalization of healthcare. I have several experiences, such as internships and competitions, regarding Web Development and Data Science / Machine Learning

Currently, I have started drafting my proposal and I would be more than happy if you could guide me on making the draft proposal. Also, I was wondering whether I could ask for mentors' feedback before submitting to the GSoC platform?

Looking forward to hearing from you soon and working with the community and fellow contributors!

---
Here is my profile
GitHub: http://github.com/malikrafsan
Personal Website: https://site.malikrafsan.tech/
Email: malikakbarrafsan@gmail.com
Resume: https://drive.google.com/file/d/1Y6hbJb7PhFKJV4yw4BVGk_ZUowybpsZe/view
